### ISSUE TITLE
Github issues: lock closed issues which continued in a new issue 

### ISSUE THREAD

**Issue by Xerkus** [docker/docker#21554]
I find it is extremely hard to follow issues in this project.
Issues are getting closed as they grow too big and continued in new ones, sometime they turn into alternative approaches or split. Problem is, discussions in original issues are not stopping, they continue for months and in some cases for years. It makes incredibly hard to track what is going on, what is considered and discarded, postponed or even actually implemented and merged half an year ago.

I propose that such issues will be locked as soon as new issue for the topic is started to prevent fragmentation and need to cross-track all of them.

**Comment by thaJeztah** [comment]
Yup, we should use this more often!
